### ISSUE TITLE
Subtle Game recordings 'flicker' when replayed

### ISSUE THREAD

**Issue by rhoslynroebuck** [IRL2/SubtleGame#198]
There is a flickering when replaying subtle game recordings that doesn't happen with NanoVer imd recordings.

**Comment by rhoslynroebuck** [comment]
From testing with Jonathan, we realised this is to do with the visualisations. When testing out recording with the rust server current in the subtle game repo + NanoverIMD, the recordings only start to flicker when replaying a state in which the visualisation has been changed. 
We tested the Subtle Game nanotube sim + setting the gradient visualisation (identical as to the puppeteering client) using a python client/ Replaying this caused flickering as soon as the visualisation was set in the shared state. Replaying only the trajectory had no issues, even when we set the same gradient visualisation on-the-fly using the same python client.
